### DISCUSSION TITLE
Kapı arkasındaki kediler ve aslanlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#225] [Zeka Sorusu]
Beş kapıdan her birinin ardında ya bir aslan ya da bir kedi bulunmaktadır. Beş hayvandan en az ikisinin kedi olduğunu biliyoruz. Her bir kapının üzerinde, aşağıda gösterildiği gibi bir yazı asılıdır. Kapının ardında aslan varsa yazılan yanlış, kedi varsa doğrudur. Bir kediyle karşılaşmayı garanti etmek için hangi kapıyı açmalısınız?
Kapı I : Burada III 'dekinden farklı tür bulunmaktadır.
Kapı II : Üç aslan vardır.
Kapı III : Üç kedi vardır.
Kapı IV : Burada kedi vardır.
Kapı V : Yegane kedi buradadır.

**Comment by Adaisland** [comment]
1.kapı(emin değiliz)
2.kapı(kedi) iki kedi olduğundan emin olduğumuz için 5-2 den 3 aslan olması doğru bir yargı
3.kapı(aslan) 2 kedi olduğunu zaten bildiğimiz için yanlış bir yargı
4.kapı(emin değiliz)
5.kapı(aslan)tek kedi yok

Cevap:2.kapı açılmalı
